styles.css

```diff
@@ -68,7 +68,7 @@ button:active {
   width: 100%;
   align-items: center;
   padding: 45px;
-  background-image: url("/Images/header_illustration.png");
+  background-image: url("Images/header_illustration.png");
```

Comment:
Some of the images are missing in the live version of the project, kindly fix these issues as this is a requirement that the online version works properly😉.

<img width="572" alt="image" src="https://github.com/Arch-Noize/portfolio-microverse/assets/64643995/f2a0a0d2-4ab3-497d-bcae-b91192ab1718">